be/src/storage/rowset/index_page.cpp

```diff
@@ -115,11 +115,12 @@ Status IndexPageIterator::seek_at_or_before(const Slice& search_key) {
 
 Status IndexPageIterator::seek_at_or_after(const Slice& search_key) {
     size_t num_entries = _reader->count();
```

Comment:
It's better to add an annotation about this function to differentiate the seek the point and the range.